Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.m

```diff
@@ -0,0 +1,23 @@
+#import "QuickSpec+QuickSpec_MethodList.h"
+#import <objc/runtime.h>
+
+
+@implementation QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors {
+    id t = [[[self class] alloc] init];
+    NSMutableSet<NSString*> *allSelectors = [NSMutableSet set];
+    
+    unsigned int methodCount = 0;
+    Method * mlist = class_copyMethodList(object_getClass(t), &methodCount);
```

Comment:
Nit pick: Remove the space between `Method *` and `mlist`. It's more consistent with the spacing in the rest of this file, and the rest of the codebase.